spm_tutorials/spm-FaceRepParametric_smdl.json

```diff
@@ -0,0 +1,58 @@
+{
+    "BIDSModelVersion": "1.0.0",
+    "Name": "SPM_FaceRepetitionParametric",
+    "Description": "Look at effect of 2nd order parametric modulation",
+    "Input": {
+        "Include": {
+            "task": "FaceRepetition"
+        }
+    },
+    "Nodes": [
+        {
+            "Level": "subject",
+            "Model": {
+                "X": [
+                    "trial_type.nonfamous_1",
```

Comment:
As was discussed in the last meeting, I am adding here the "provenance" of some of those predictors.

Not sure if we want to make this a requirement in the specs, but this would simplify implmentation in any case.